common/app/routes/Challenges/Solution-Input.jsx

```diff
@@ -22,9 +23,16 @@ export default function SolutionInput({ solution, placeholder }) {
         { ...DOMOnlyProps(solution) }
       />
       {
-        validationState === 'error' ?
-          <HelpBlock>Make sure you provide a proper URL.</HelpBlock> :
-          null
+        validationState === 'error' &&
+          <HelpBlock>Make sure you provide a proper URL.</HelpBlock>
+      }
+      {
+        validationState === 'warning' &&
+          <HelpBlock>
+            If you are using Glitch, make sure you have entered a shareable URL
```

Comment:
~If you are using Glitch, m~ake sure you have entered a shareable URL

Make sure you have entered a shareable URL

---
We know the user is using glitch if we have got this far.